com.unity.netcode.gameobjects/Runtime/Transports/UTP/SecureAccessor.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace Unity.Netcode.Transports.UTP
+{
+    public class SecureAccessor : MonoBehaviour
+    {
+        private void Awake()
+        {
+            var serverSecrets = new ServerSecrets();
+
+            try
+            {
+                serverSecrets.ServerCertificate = ServerCertificate;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            try
+            {
+                serverSecrets.ServerPrivate = ServerPrivate;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            var clientSecrets = new ClientSecrets();
+            try
+            {
+                clientSecrets.ClientCertificate = ClientCA;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            try
+            {
+                clientSecrets.ServerCommonName = ServerCommonName;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            GetComponent<UnityTransport>().SetServerSecrets(serverSecrets);
```

Comment:
What happens if someone adds this component when they're using a different transport?

Obviously we don't support that, but feels like an editor component should be added to prevent it from being added at all in that case. Otherwise the only indication it won't work is a NullReferenceException.

Reply:
I can add a null check and log a meaningful error message